react-native/android/app/build.gradle

```diff
@@ -79,12 +89,24 @@ android {
         applicationId "io.keybase.android"
         minSdkVersion 16
         targetSdkVersion 23
-        versionCode 2
-        versionName "1.0.0-alpha-1"
+        // Marco messed up the version code originally, so now we have to set the 2nd and 3rd msb
+        // The top one would make the number negative
+        versionCode (timestamp() | (1 << 30) | (1 << 29))
```

Comment:
I messed this up while testing something, and forced us into a really big version code. I've emailed support asking if they could just delete them, but I can't delete those versions of the apk myself.